package.json

```diff
@@ -24,7 +24,7 @@
   "scripts": {
     "build": "turbo run build",
     "dev": "turbo run dev",
-    "test": "turbo run test",
+    "test": "turbo run test --",
```

Comment:
What `--` does?

Reply:
it allows passing args to the subpackage commands.

in this case it will allow running `yarn test --filter=alert` from the monorepo root